app/controllers/organizers/teams_controller.rb

```diff
@@ -12,7 +12,12 @@ def index
       else
         @teams = Team.order(:kind, :name)
       end
-      if params[:filter] != 'all'
+
+      if params[:filter] == 'full_time'
+        @teams = Team.in_current_season.full_time.ordered
+      elsif params[:filter] == 'part_time'
+        @teams = Team.in_current_season.part_time.ordered
+      elsif params[:filter] != 'all'
         @teams = Team.in_current_season.selected.ordered
```

Comment:
Hm, why do we even have the `selected` scope around any more? Isn't this the same as `accepted`?